### DISCUSSION TITLE
Bridge support

### DISCUSSION THREAD

**Discussion by dgduncan** [mochi-mqtt/server#129] [General]
Just wanted to ask if there is currently broker bridge support if it is in consideration for the roadmap.

**Comment by torntrousers** [comment]
May I gently wonder if its good manors to keep posting about your own project on another projects issue tracker?

**Comment by wind-c** [comment]
@torntrousers You have a point, so I deleted the above post. I am also a contributor to mochi-co/mqtt.
@dgduncan The bridge is easier to implement through hooks.
